workers/discourse_analysis_worker/trained_crf_model/directory.md

```diff
@@ -0,0 +1 @@
+For trained models
```

Comment:
Why is this file here?

Reply:
@ccarterlandis Its a GitHub "trick". If a directory is empty, it will not push it to the repository. However, we need this directory because that's where the models are generated. A better solution would be a mkdir command in the worker, however the risk there is installation on a file system where the user has restricted rights for making directories. We have 2 secure installations where this is the case.

Reply:
fixed with the .gitkeep file @aksh555 mentioned in the end. @ccarterlandis